### PR TITLE
Fix NoMethodErrors in GIS Expo form

### LINKED ISSUE #219: NoMethodError in Contribute#new (GIS Expo Student Winners)
**STEPS TO REPRODUCE**
1. Log in to the application
1. Visit /contribute
1. Select "GIS Expo Student Winners"
1. Click "Begin"

**ERROR DETAILS**
```
NoMethodError in Contribute#new
Showing /Users/mark/Documents/workspace/_no_backup_/RubyMine_Projects/epigaea/app/views/contribute/deposit_view/_gis_poster.html.erb where line #6 raised:

private method `title' called for #<GisPoster:0x007fd21cae7248>
Did you mean?  title=
Extracted source (around line #6):
4
5
6
7
8
9
              


<%= f.text_field :title, label: 'Title', help_block: 'Please limit titles to 250 characters or less.', required: 'required' %>

<%= f.text_field :creator, value: current_user.display_name, label: 'Contributor', required: 'required'  %>
<div class="checkbox_container">

Trace of template inclusion: app/views/contribute/new.html.erb
```

### PR DESCRIPTION
**Pull request by little9**
Closes #219

### CODE REVIEW

File: app/models/forms/contribution.rb [hunk 1]
```diff
@@ -84,11 +84,12 @@ def insert_embargo_date
     @tufts_pdf.embargo_note = (Time.zone.now + embargo_note.to_i.months).iso8601 unless (embargo_note || '0').eql? '0'
   end
 
-  def insert_rels_ext_relationships
-    return unless @tufts_pdf
-    @tufts_pdf.stored_collection_id = parent_collection
-    @tufts_pdf.rels_ext.serialize!
-  end
+  # def insert_rels_ext_relationships
```
Comment (no-reply) [inline app/models/forms/contribution.rb]: Is there a ticket for this todo? If so, can we just drop this code?

File: app/models/forms/gis_poster.rb [hunk 2]
```diff
@@ -6,24 +6,23 @@ class GisPoster < GenericTischDeposit
                      :bibliographic_citation, :subject, :corpname, :attachment, :license]
   # def copy_attributes
   #  super
-  #  @tufts_pdf.description = ["#{description}  Submitted in partial fulfillment of the grant requirement of the Tufts Summer Scholars Program."]
+  #  @tufts_pdXFf.description = ["#{description}  Submitted in partial fulfillment of the grant requirement of the Tufts Summer Scholars Program."]
```
Comment (no-reply) [inline app/models/forms/gis_poster.rb]: Typo?

File: app/models/forms/contribution.rb (hunk 1 above)
Comment (little9) [reply app/models/forms/contribution.rb]: I made issue #256 for this, I'll just drop this method from the code

File: app/models/forms/gis_poster.rb (hunk 2 above)
Comment (little9) [reply app/models/forms/gis_poster.rb]: Yeah, I must have been trying in vain to open a file... `C-x C-f`

**Pull request merged**